builder/builder.go

```diff
@@ -743,3 +750,23 @@ func (b *Builder) BuildDeltaManifestsPreviousVersions(prev, to uint32) error {
 
 	return nil
 }
+
+func (b *Builder) signLatestVersion(formatdir string) error {
```

Comment:
other than the file name, this is an exact copy of ` SignManifestMoM()`. Instead of duplicating the code, create a function that can be used for both. Also, keep the code in `update.go`.